README.md

```diff
@@ -245,3 +246,7 @@ First, please read the [contributing docs](CONTRIBUTING.md). Generally, you will
 ## Adding new sbatch scripts
 
 You can add more sbatch scripts by putting them in the sbatches directory.
+=======
+# FarmshareJupyter
```

Comment:
Not sure what happened to the README but this first line and bottom here look like a merge conflict not resolved.